src/Pomelo.EntityFrameworkCore.MySql/Extensions/MySqlDbContextOptionsExtensions.cs

```diff
@@ -87,5 +90,25 @@ private static MySqlOptionsExtension GetOrCreateExtension(DbContextOptionsBuilde
                 ? new MySqlOptionsExtension(existing)
                 : new MySqlOptionsExtension();
         }
+
+        private static bool IsAbleToUseDateTime6(string connectionStr)
```

Comment:
Now we have an issue, when you `dotnet ef migrations add xxx`, the ef will lost the extension, so the UseDateTime6 and TreatTinyAsBoolean will be false. That is why the ci fails